Components/Pages/Localities/Index.razor

```diff
@@ -68,46 +68,48 @@ else
-            @for (int i = pageNumber; i <= pageCount; i++)
-            {   
-                // Não sei por que, mas quando clica no número em um cenário especifico ele diz que o "i" é um número maior que o pageCount
-                // Pesquisar por "Santo"
-                // Ir até a página 5
-                // Clicar no número 7, por exemplo
-                // Ele chega no breakPoint com o i == 9 e o meu pageCount == 8
-                @* <li class="page-item @(pageNumber == i ? "active" : "")">
-                    <button class="page-link" @onclick="e => Search(i)">@i</button>
-                </li> *@
-
-                @* @if (isLimiteVisualizacao(i) && canMostrarUltimasPaginas)
-                {
-                    <li class="page-item">
-                        <span class="page-link">...</span>
-                    </li>
-                       
-                    <li class="page-item">
-                        <button class="page-link" @onclick="e => Search(pageCount)">@pageCount</button> 
-                    </li>
 
-                    break;
-                } *@
+            @*
+        @for (int i = pageNumber; i <= pageCount; i++)
+        {
+        // Não sei por que, mas quando clica no número em um cenário especifico ele diz que o "i" é um número maior que
+        o pageCount
+        // Pesquisar por "Santo"
+        // Ir até a página 5
+        // Clicar no número 7, por exemplo
+        // Ele chega no breakPoint com o i == 9 e o meu pageCount == 8
+        @* <li class="page-item @(pageNumber == i ? "active" : "")">
+        <button class="page-link" @onclick="e => Search(i)">@i</button>
+        </li> *@
+
+            @* @if (isLimiteVisualizacao(i) && canMostrarUltimasPaginas)
+        {
+        <li class="page-item">
+        <span class="page-link">...</span>
+        </li>
+
+        <li class="page-item">
+        <button class="page-link" @onclick="e => Search(pageCount)">@pageCount</button>
+        </li>
+
+        break;
+        } *@
             @* } *@
             <li class="page-item @(hasMoreRecords ? "" : "disabled")">
                 <button class="page-link" @onclick="e => Search(pageNumber + 1)">Próximo</button>
             </li>
         </ul>
     </nav>
-} 
+}
 
 @code {
     [Parameter]
     public int pageNumber { get; set; } = 1;
     private bool isFirstPage => pageNumber <= 1;
-    private const int maxPaginasVisiveisPaginacao = 2;
-    private bool isLimiteVisualizacao(int paginaAtual) => paginaAtual >= pageNumber + maxPaginasVisiveisPaginacao;
-    private bool canMostrarUltimasPaginas => pageCount - maxPaginasVisiveisPaginacao >= pageNumber + maxPaginasVisiveisPaginacao;
+    private const int maxVisiblePagesPagination = 2;
+    private bool isLimiteVisualizacao(int currentPage) => currentPage >= pageNumber + maxVisiblePagesPagination;
+    private bool canMostrarUltimasPaginas => pageCount - maxVisiblePagesPagination >= pageNumber +
+    maxVisiblePagesPagination;
```

Comment:
Mesma situação da sugestão anterior, aqui também ficou em português, mas não vejo problema visto que o padrão da chamada "can/get/set/is/max" estaria padronizado em (Ação do que vai fazer) em ingles  seguido da Descrição.